hr_timesheet_analysis/views/account_analytic_line.xml

```diff
@@ -4,17 +4,6 @@
     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-    <record id="hr_timesheet_line_pivot" model="ir.ui.view">
```

Comment:
Pivots are gone from v14?

Reply:
It is alreday available in 14 core so removed it.